aws_lambda_builders/workflows/python_pip/validator.py

```diff
@@ -0,0 +1,65 @@
+"""
+Python Runtime Validation
+"""
+
+import logging
+import os
+import subprocess
+
+from aws_lambda_builders.exceptions import MisMatchRuntimeError
+
+LOG = logging.getLogger(__name__)
+
+
+class PythonRuntimeValidator(object):
+    SUPPORTED_RUNTIMES = [
+        "python2.7",
+        "python3.6",
+        "python3.7"
+    ]
+
+    def __init__(self, runtime, runtime_path):
+        self.language = "python"
+        self.runtime = runtime
+        self.runtime_path = runtime_path
+
+    def has_runtime(self):
+        """
+        Checks if the runtime is supported.
+        :param string runtime: Runtime to check
+        :return bool: True, if the runtime is supported.
+        """
+        return self.runtime in self.SUPPORTED_RUNTIMES
```

Comment:
Sets are better for this comparison: https://stackoverflow.com/questions/2831212/python-sets-vs-lists